src/mlpack/tests/test_tools.hpp

```diff
@@ -48,6 +48,21 @@ inline void CheckMatrices(const arma::Mat<size_t>& a,
     BOOST_REQUIRE_EQUAL(a[i], b[i]);
 }
 
+template <class fieldType,
+          class = std::enable_if_t<
+              arma::is_Col<typename fieldType::object_type>::value ||
+              arma::is_Mat_only<typename fieldType::object_type>::value>>
+// Check the values of two unsigned matrices.
```

Comment:
I don't think that they have to be unsigned?  Also these are fields, not matrices, right?

Reply:
Yes, sorry I forgot to change the comment when copying the ```checkmatrice``` code